### DISCUSSION TITLE
Running backup:list freezes & takes a long time

### DISCUSSION THREAD

**Discussion by kendepelchin** [spatie/laravel-backup#1666] [Q&A]
When I run the command `php artisan backup:list` using a Dropbox filesystem, the command is stuck and doesn't give me any output. I don't see any logs in php or laravel as well. I'm wondering what this issue could be?

When I run `backup:run` the zip file is correctly created and stored on the Dropbox account. But after the slack notifications, the command gets stuck as well.

Eventually, when the commands run for 10', I get the output `The backups on the dropbox disk are considered healthy.` 🤕

I'm not sure where I should look now. Any help would be greatly appreciated.

**Comment by parallels999** [comment]
seems like dropbox flysystem problem
